### DISCUSSION TITLE
KBV Profile bei Rezeptanforderung erfordern Pflichtangaben, die ein AVS evtl. nicht hat.

### DISCUSSION THREAD

**Discussion by bNaumann-gem** [gematik/spec-E-Rezept-ServiceRequest#78] [Q&A] [answered]
Aktuell ist bei der Rezeptanforderung u.a. der Practitioner und der Patient anzugeben. Beide Inhalte erfordern gewisse Pflichtfelder. Bei der Rezeptanforderung hat die Apotheke unter Umständen aber nicht alle Infos/Daten parat, um eine FHIR-valide! Anforderung zu erstellen.
Bsp:
- Patient wird nur über KVNR identifiziert, Name / Adresse / birthDate ist unbekannt
- Arzt wird nur anhand der LANR identifiziert, qualification-typ & -Berufsbezeichnung ist unbekannt.

Das annehmende KIS könnte die die Anforderung auch mit weniger Daten verarbeiten. Der Datensatz ist aber nicht FHIRvalide.

Kann hier eine Empfehlung abgegeben werden, wie sich ein AVS verhalten soll, wenn Daten nicht bekannt sind, aber verpflichtend anzugeben sind, bspw. über Dummydaten?
@florianschoffke @SvenSommer

**Comment by florianschoffke** [comment, marked as answer]
Im Profil [Prescription Request](https://simplifier.net/erezept-servicerequest/erp-service-request-prescription-request) sind die beiden Felder .subject und .performer zwar Pflichtangaben, referenzieren technisch aber auf die "generischen" FHIR-Ressourcen "[Patient](https://hl7.org/fhir/R4/patient.html)" und "[Practitioner](https://hl7.org/fhir/R4/practitioner.html)".
Dort sind fast alle Angaben optional.
Es muss aber auch nicht mal ein weiteres Objekt mitgegeben werden, es ist theoretisch auch möglich
_ServiceRequest.subject.reference.display = "Klaus Müller"_ anzugeben.

Die Profilierung ist also sehr offen gehalten, um diese Fälle abzudecken.
